docs/resources/cadence_cluster_v2.md

```diff
@@ -64,6 +65,9 @@ The following terms are used to describe attributes in the schema of this resour
 *___sla_tier___*<br>
 <ins>Type</ins>: string, required, immutable<br>
 <ins>Constraints</ins>: allowed values: [ `PRODUCTION`, `NON_PRODUCTION` ]<br><br>SLA Tier of the cluster. Non-production clusters may receive lower priority support and reduced SLAs. Production tier is not available when using Developer class nodes. See [SLA Tier](https://www.instaclustr.com/support/documentation/useful-information/sla-tier/) for more information.<br><br>
+*___use_http_api___*<br>
+<ins>Type</ins>: boolean, required, immutable<br>
```

Comment:
Should this be a required setting? Or optional and if absent we treat as false?
Check with #team-customerplatform about what should be the best practice.

Reply:
I'll check with them

Reply:
this is better be optional with default false

adding new `required` property means breaking compatibility and we'll need to bump the API endpoint version